e2e-tests/IIoTPlatform-E2E-Tests/Extensions/IConfigurationBuilderEx.cs

```diff
@@ -18,26 +18,26 @@ namespace IIoTPlatform_E2E_Tests.Extensions {
     /// </summary>
     public static class IConfigurationBuilderEx {
 
-        /// <summary>
-        /// Add environment variables from a specific environment variable target.
-        /// </summary>
-        /// <param name="configurationBuilder"></param>
-        /// <param name="environmentVariableTarget"></param>
-        /// <returns></returns>
-        public static IConfigurationBuilder AddEnvironmentVariables(
-            this IConfigurationBuilder configurationBuilder,
-            EnvironmentVariableTarget environmentVariableTarget
-        ) {
-            var envVarDict = Environment.GetEnvironmentVariables(environmentVariableTarget);
+        ///// <summary>
```

Comment:
why it this code commented out?

Reply:
code removed. Because now 
`public static IConfigurationBuilder AddEnvironmentVariables(
            this IConfigurationBuilder configurationBuilder,
            EnvironmentVariableTarget environmentVariableTarget)` is defined in  `Microsoft.Azure.IIoT.Core`